src/handlers/scheduledEventHandler.ts

```diff
@@ -94,3 +95,49 @@ export const addMissedUpdatesRole = async (env: env) => {
 		console.error('Error while adding missed updates roles');
```

Comment:
do we really need this..?(or is this some to store log)

Reply:
This would help us with debugging on prod if this particular API is failing